.gitignore

```diff
@@ -14,3 +14,6 @@ server/manifest.go
 
 # notice
 .notice-work
+
+# Aider
+.aider*
```

Comment:
0/5, but it feels like a lot of effort to have to propagate all these things across our many repos. Would you consider adding the following to your `~/.gitconfig`:

```
[core]
	excludesfile = ~/.gitignore_global
```

and then adding this things there instead?